platform/qt/src/qmapbox.cpp

```diff
@@ -24,16 +24,215 @@ static_assert(mbgl::underlying_type(QMapbox::Feature::PolygonType) == mbgl::unde
 
 namespace QMapbox {
 
+/*!
+    \namespace QMapbox
+    \inmodule Mapbox Qt SDK
+
+    Contains miscellaneous Mapbox bindings used throughout QMapboxGL.
+*/
+
+/*!
+    \typedef QMapbox::Coordinate
+
+    Synonim for QPair<double, double>.
+    Reflects mbgl::Point<double>.
+*/
+
+/*!
+    \typedef QMapbox::CoordinateZoom
+
+    Synonim for QPair<Coordinate, double>.
+    Used as return value in QMapboxGL::coordinateZoomForBounds.
+*/
+
+/*!
+    \typedef QMapbox::Coordinates
+
+    Synonim for QList<QMapbox::Coordinate>.
+    Reflects mbgl::{LineString,LinearRing,MultiPoint}<double>.
```

Comment:
Not sure if we should mention this. The point of the binding is to hide the core implementation.